File: C/bee 1012.c
#include <stdio.h>

int main(void) {

    double a,b,c;
    scanf("%lf %lf %lf", &a, &b, &c);


    printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n", ((a * c)/2), (3.14159 * c*c), ((a + b) * c / 2), (b*b), (a*b));

    return 0;
}

File: Lua/bee 1012.lua
local a = tonumber(io.read("*n"))
local b = tonumber(io.read("*n"))
local c = tonumber(io.read("*n"))

print(string.format("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f", ((a * c)/2), (3.14159 * c^2), ((a + b) * c / 2), (b^2), (a*b)))
